tests/TestCase/View/Helper/PaginatorHelperTest.php

```diff
@@ -50,6 +59,7 @@ public function setUp()
         Router::connect('/:controller/:action/*');
         Router::connect('/:plugin/:controller/:action/*');
 
+        // @todo Set locale for unit test without using private property.
```

Comment:
Todos almost always get left forever :)  Either fix the issue or remove the todo.